control/init.sh

```diff
@@ -0,0 +1,61 @@
+#! /bin/bash
+if ! hash virtualenvwrapper.sh 2>/dev/null; then
+    echo "Please install virtualenvwrapper and make sure it is in your PATH"
+    echo ""
+    echo "  sudo apt-get install git python-dev python-pip"
+    echo "  sudo pip install virtualenv virtualenvwrapper"
+    return 1
+fi
+
+source virtualenvwrapper.sh
+
+if [ ! -d ~/.virtualenvs/ansible ]; then
+    echo "Creating ansible virtualenv..."
+    mkvirtualenv ansible
+    pip install -r ~/commcarehq-ansible/ansible/requirements.txt
+else
+    workon ansible
+fi
+
+if ! [ -L ~/commcarehq-ansible/config -o -d ~/commcarehq-ansible/config ]; then
+    # TODO setup permissions (add user to group?) to allow write to config repo
+    #su ansible -c "usermod -a -G ansible $USER"
+    ln -s /home/ansible/commcarehq-ansible/config ~/commcarehq-ansible/config
+fi
+
+if [ ! -d ~/commcare-hq-deploy ]; then
+    echo "Cloning commcare-hq-deploy..."
+    git clone git@github.com:dimagi/commcare-hq-deploy ~/commcare-hq-deploy
+    if [ ! -d ~/commcare-hq ]; then
+        # keep old commands working: create symlink to simulate cchq repo
+        mkdir ~/commcare-hq
+        ln -s ~/commcare-hq-deploy/fab ~/commcare-hq/fab
```

Comment:
I think this was the only thing that was used by the deploy, so no need to have a clone of the cchq repo for each dev since inventories have moved to a separate deploy repo. Is that correct?